fusor-ember-cli/app/templates/components/text-f.hbs

```diff
@@ -3,7 +3,7 @@
   {{input class="form-control" value=value placeholder=placeholder type=typeInput focus-out="showValidationErrors" id=cssId disabled=disabled autocomplete='off'
   maxlength="250"}}
 
-  {{#if isPassword}}
+  {{#if canShowPassword}}
           <i {{action 'showPassword'}} class="fa {{eyeIcon}} eye-icon"></i>
   {{/if}}
```

Comment:
Yeah.  I can totally see that coming back on the password entry fields (where it's just an entry, not a password set, and there's no confirmation)